from django.utils.translation import gettext_lazy as _

UNKNOWN_ERROR = _("An unexpected error occurred. Please try again later.")
